Applications/Spire/Source/Ui/TextBox.cpp

```diff
@@ -513,6 +513,7 @@ void TextBox::resizeEvent(QResizeEvent* event) {
 void TextBox::showEvent(QShowEvent* event) {
   if(!m_line_edit && (!is_read_only() || isEnabled())) {
     initialize_line_edit();
+    m_line_edit->show();
```

Comment:
Provide a brief explanation for what this fixes.

Reply:
This change is unnecessary. I will restore it.